core/src/avm2/globals/flash/media/Video.as

```diff
@@ -0,0 +1,19 @@
+package flash.media {
+    public final class Video {
+        private var _videoWidth: int;
+        private var _videoHeight: int;
+
+        public function Video(videoWidth:int = 320, videoHeight:int = 240) {
```

Comment:
let's add a stub mention to the constructor

Reply:
I think Adrian meant like https://github.com/ruffle-rs/ruffle/blob/master/core/src/avm2/globals/flash/utils/Dictionary.as#L10

Reply:
`Video` doesn't actually do anything itself; it only serves as a container to be filled by `Video.attachCamera` or `Video.attachNetStream`. Adding a stub warning will unnecessarily warn about the Video stubbed on content that doesn't use `attachNetStream`, in which case the `Video` container is functioning correctly.